controller/main_controller.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ *
+ * Collapsible Categories extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2015 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\collapsiblecategories\controller;
+
+class controller implements main_interface
+{
+	/** @var \phpbb\request\request */
+	protected $request;
```

Comment:
We prefer to order these alphabetically, so they should be reveresed here, and in the constructor and constr. args and in the service file ;)